docs/pages/careers/staff-ui-engineer-base-ui.md

```diff
@@ -0,0 +1,94 @@
+# Staff UI Engineer — Base UI
+
+<p class="description">Research, build, document, and ship high-quality, unstyled UI components with a focus on a11y.</p>
+
+## Details of the role
+
+- **Location**: Remote (preference for UTC-6 to UTC+5).
```

Comment:
**Off-topic**

I was wondering if we should consider extending [the default template](https://github.com/mui/material-ui/blob/master/docs/pages/careers/ROLE_TEMPLATE.md) to:

```suggestion
- **Location**: Remote (preference for UTC-6 to UTC+6).
```

India is UTC +5.5 but is the largest developer pool, GDP is growing but it's also not amazing https://docs.google.com/spreadsheets/d/1rJ6rUdlj8uNPt3nLLY9KkrDiTf3tL8YW5rNgV_iyYgE/edit#gid=0.

I guess we can keep it as is.